source/documentation/modules.html.md.erb

```diff
@@ -0,0 +1,282 @@
+---
+title: Built-In Modules
+introduction: >
+  Sass provides many built-in modules which contain useful functions (and the
+  occasional mixin). These modules can be loaded with the
+  [`@use` rule](at-rules/use) like any user-defined stylesheet, and their
+  functions can be called [like any other module
+  member](at-rules/use#loading-members). All built-in module URLs begin with
+  `sass:` to indicate that they're part of Sass itself.
+overview: true
+---
+
+<% content_for :before_introduction do %>
+  <%= partial 'snippets/built-in-module-status' %>
+<% end %>
+
+<% heads_up do %>
+  Before the Sass module system was introduced, all Sass functions were globally
+  available at all times. Many functions still have global aliases (these are
+  listed in their documentation). The Sass team discourages their use and will
+  eventually deprecate them, but for now they're still available for
+  compatibility with older Sass versions and with LibSass (which doesn't support
+  the module system yet).
+
+  [A few functions][] are still *only* available globally, either because they
+  have special evaluation behavior ([`if()`][]) or because they add extra
+  behavior on top of built-in CSS functions ([`rgb()`][] and [`hsl()`][]). These
+  will not be deprecated and can be used freely.
+
+  [a few functions]: #global-functions
+  [`if()`]: #if
+  [`rgb()`]: #rgb
+  [`hsl()`]: #hsl
+<% end %>
+
+<% example do %>
+  @use "sass:color";
+
+  .button {
+    $primary-color: #6b717f;
+    color: $primary-color;
+    border: 1px solid color.scale($primary-color, $lightness: 20%);
+  }
+  ===
+  @use "sass:color"
+
+  .button
+    $primary-color: #6b717f
+    color: $primary-color
+    border: 1px solid color.scale($primary-color, $lightness: 20%)
+  ===
+  .button {
+    color: #6b717f;
+    border: 1px solid #878d9a;
+  }
+<% end %>
+
+Sass provides the following built-in modules:
+
+* Any function that Sass doesn't recognize as built-in or user-defined is
```

Comment:
Move this bullet to the bottom of this list until there's a better place?

Reply:
Oh, I just meant to delete this when I moved the plain CSS stuff into the `@function` documentation.